src/component/gas_production_component.cc

```diff
@@ -14,14 +14,14 @@ gas_production_component_manager::create_component_instance_data(unsigned count)
     size = sizeof(unsigned) * count + align_size<unsigned>(size);
```

Comment:
notice that here we are overwriting the existing value, rather than adding to it on this and the following lines.